portal/assets/themes/unsplash/custom_portal.css

```diff
@@ -0,0 +1,62 @@
+/*
+===============================================================================
+ This file contain extra CSS rules to customize the YunoHost user portal and
+ can be used to customize app tiles, buttons, etc...
+===============================================================================
+*/
+
+/* Make page texts white */
+.user-container h2,
+.user-container small,
+.user-container .user-mail,
+.user-container .user-mail,
+.content .footer a,
+a.app-tile,
+#ynh-logout {
+  color: white !important;
+}
+
+body {
+  color: white !important;
+  text-shadow: 3px 4px 4px rgba(0,0,0,.4), -1px -1px 6px rgba(0,0,0,0.2);
+}
+
+.ynh-user-portal {
+  background-image: url('https://source.unsplash.com/random/featured/?nature') !important;
```

Comment:
Could be interesting to say that this theme will load an external image each time, I'm expecting some people to don't want to that for privacy reason '-'